README.md

```diff
@@ -1,193 +1,198 @@
 # The New Stack (TNS) observability app
 
```

Comment:
Pls, add the TOC here. It adds structure to the readme. Thanks.

Reply:
GitHub autogenerates a TOC from the markdown heading and keeps it in sync when things change. Do we prefer to keep a copy of the TOC in the body of the document as well? Looking at other examples of documentation (https://github.com/grafana/mimir, https://github.com/grafana/agent/tree/main/example/k3d) it doesn't seem common practice, but happy to add it back if it is preferred.